### DISCUSSION TITLE
Missing VTK dll's

### DISCUSSION THREAD

**Discussion by KennFischerAmfitech** [PlusToolkit/PlusLib#1176] [Q&A]
Hi

I'm in the process of adding a new device to the toolkit. (Trying to build it without my change)
I have everything build successfully, but when i run the PlusServerLauncher.exe in the bin/debug folder or try to debug the server launcher from PlusApp-bin it says "vtkCommonDataModel-9.1d.dll" "vtkIOXMLParser-9.1d.dll" "vtkCommonSystem-9.1d.dll" and "vtkCommonCore-9.1d.dll" was not found. 
I can see that they are located in the vtk-bin/bin/debug folder. 
So my question is how do i make the project use those, so i can try to debug on it?

Best regard Kenn

**Comment by kenavolic** [comment]
Add the path to the dll directory to the windows path environment variable
